locales/languages.json

```diff
@@ -64,6 +64,9 @@
     },
     "NL": {
       "name": "Netherlands"
+    },
+    "CH": {
+      "name": "Switzerland"
```

Comment:
Does they use the English name for their country ?

Reply:
it's translated.
that's the reason why it'added in localization file